.github/workflows/status-embed.yml

```diff
@@ -0,0 +1,76 @@
+name: Status Embed
+
+on:
+  workflow_run:
+    workflows:
+      - Event Validation
+    types:
+      - completed
+
+jobs:
+  status_embed:
+    # We need to send a status embed whenever the workflow
+    # sequence we're running terminates. There are a number
+    # of situations in which that happens:
+    #
+    # 1. We reach the end of the Deploy workflow, without
```

Comment:
Not critical, but maybe the name should be changed here also?

Reply:
Indeedly, fixed in 8d3e82c3dfee4bc2c1f5942cc54a2bb72406edd2, thanks!